woocommerce/woocommerce-ios
# [Stats] Remove unused `price` property from `TopEarnerStatsItem` storage entity

Closes: #11485 
⚠️ Depends on #11491 ⚠️ 


## Why/How
In #11491 we removed an unused `price` property from `TopEarnerStatsItem`. This PR removes that `price` attribute from the corresponding storage entity.

These changes include a new Core Data model and migration test for the change.

## Testing instructions


Optional testing:

1. Build and run the app.
2. Confirm there are no core data errors and the dashboard still loads, including the Top Performers section.


---
